src/functions/events/file-error/handler.ts

```diff
@@ -34,7 +34,10 @@ const sendErrorToDestination = async (event: CoreFileError) => {
   const processDeliveriesInput: ProcessDeliveriesInput = {
     destinations: errorDestinations.destinations,
     payload: event,
-    destinationFilename: `${event.detail.fileId}-${new Date().toUTCString()}`,
+    payloadMetadata: {
+      payloadId: `${event.detail.fileId}-${new Date().toUTCString()}`,
```

Comment:
We screwed up here, it needs to be `toISOString`, utc string will have spaces and commas in it and screw things up.